### ISSUE TITLE
ARIA: Attribute " aria-label ", required for accessibility, is missing on node

### ISSUE THREAD

**Issue by yash-sharma9091** [fraywing/textAngular#1446]
when using with angular material it shows the warning about aria label
``ARIA: Attribute " aria-label ", required for accessibility, is missing on node: ``
```
<textarea id="taHtmlElement8316965102261715" ng-show="showHtml" ta-bind="ta-bind" ng-model="html" class="ng-pristine ng-untouched ng-valid ng-scope md-input ta-bind ta-html ta-editor form-control ng-empty ng-hide" rows="1" aria-hidden="true" aria-invalid="false" style="height: 0px;"></textarea>
```
where we have to add aria attribute in textangular directive

**Comment by fareed543** [comment]
I Resolved this issue by adding the `aria-label` attribute.
`aria-label="ShowHideAccountant"`

if you have multiple fields and you want to ignore the issue in your application. Try to set `$mdAriaProvider.disableWarnings();`

```
app.config(function($mdAriaProvider) {
   // Globally disables all ARIA warnings.
   $mdAriaProvider.disableWarnings();
});
```

Solution to your problem: you have to add attribute to your input, select, Textarea tags
Example : `<textarea aria-label="Your Field Lable name"></textarea>`

**Comment by tbone849** [comment]
I got one of these warnings for each of the buttons for the editor. I added this code around line 4134 to textAngular.js and it removed the errors.

`toolElement.attr('aria-label', toolScope.name);`
